integration-tests/deployment/internal/testutil/helpers.go

```diff
@@ -0,0 +1,81 @@
+package testutil
+
+import (
+	ctfconfig "github.com/smartcontractkit/chainlink-testing-framework/lib/config"
+	"github.com/smartcontractkit/chainlink-testing-framework/lib/utils/ptr"
+	"github.com/smartcontractkit/chainlink/integration-tests/ccip-tests/testconfig"
+	chainlink_test_config "github.com/smartcontractkit/chainlink/integration-tests/testconfig"
+)
+
+func GetDefaultNewChainlinkClusterConfig() *testconfig.ChainlinkDeployment {
+	return &testconfig.ChainlinkDeployment{
+		Common: &testconfig.Node{
+			ChainlinkImage: &ctfconfig.ChainlinkImageConfig{
```

Comment:
this isn't env abstract - so shouldn't be under deployment/internal. Move to persistent?